client/layout/logged-out.jsx

```diff
@@ -105,6 +106,15 @@ const LayoutLoggedOut = ( {
 		! currentRoute.startsWith( '/log-in/webauthn' ) &&
 		! currentRoute.startsWith( '/log-in/backup' );
 
+	const isMagicLogin = currentRoute && currentRoute.startsWith( '/log-in/link' );
+
+	const isWpcomMagicLogin =
+		isMagicLogin &&
+		! isJetpackLogin &&
+		! isGravPoweredLoginPage &&
+		! isJetpackCloudOAuth2Client( oauth2Client ) &&
+		! isWooOAuth2Client( oauth2Client );
+
```

Comment:
I've updated logic to exclude applying new styles jetpack/woo login screens for now. It was suggested we go ahead and apply to Jetpack for consistency. But I've been able to find multiple versions of the Jetpack magic login screen. At least one is rendered very differently (see below). I feel like it is inevitable that these changes will end up applying like this in situations where we don't want them. So for now, I'm just going to keep a very narrow, safe aim, and try to affect just the wpcom screens.

Reply:
I did just spend some time playing around with one other way to differentiate the various login / magic login screens using [getMagicLoginCurrentView](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/selectors/get-magic-login-current-view.js), but this piece of state was inconsistently set, and in testing often led to the class/styles not being applied. So I'm going to stick with current approach for now.

Reply:
Thanks for checking those. 👍  Yeah we have too many of these, and future work should consolidate things more.